src/iexamine.cpp

```diff
@@ -2780,6 +2782,18 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
+{
+    if( source_mon->turn_next_milking < calendar::turn ) {
+        item milk( "milk", 0, 4 );
+        g->handle_liquid( milk, nullptr, 0, nullptr, nullptr, source_mon );
+        add_msg( "You milk the cow." );
```

Comment:
I don't know how to use `(add _())` but I think I may have fixed.

Didn't thought about that scenario regarding having no containers, thank you for pointing it out. 
I'm going to try to see how to make it so you can save un-used charges inside the cow.